Google.Cloud.Functions.Invoker.Tests/FunctionEnvironmentTest.cs

```diff
@@ -94,6 +94,7 @@ public async Task TargetFunction_CommandLineOverridesEnvironmentVariable()
             await AssertHttpFunctionType<DefaultFunction>(environment);
         }
 
+        // This will throw (correctly) because there are multiple function types in this assembly.
```

Comment:
nit: Quickly reading this I understood that the test was throwing. Maybe change "This will throw (correctly)" by "This is a bad environment ..."

Reply:
Yup. The comment was even worse the first time :)